### DISCUSSION TITLE
LSP4IJ 0.3.0 Release

### DISCUSSION THREAD

**Discussion by fbricon** [redhat-developer/lsp4ij#426] [Announcements]
Hi,

we are about to release LSP4IJ 0.3.0. Highlights of this release include:
 * Support for textDocument/semanticTokens
 * Support file link in hover
 
 More details in the <a href="https://github.com/redhat-developer/lsp4ij/milestone/3?closed=1">changelog</a>.
 
 Please give a try to the [latest nightly build](https://plugins.jetbrains.com/plugin/23257-lsp4ij/versions/nightly).

 Unless you report a blocking issue, we'll proceed with the release tomorrow 18/07/2024.
 
 cc @CppCXY @InSyncWithFoo @seachicken @KUGDev @AlexWeinstein92 @aparnamichael @MituuZ @turkeylurkey @mrglavas @koxudaxi @FalsePattern

**Comment by fbricon** [comment]
FTR the release job has been stuck in 'Queued' state for a while, I can't even cancel it https://github.com/redhat-developer/lsp4ij/actions/runs/10000143672.
Will get back to it after I get some shut eye

**Reply by InSyncWithFoo** [reply]
The issue seems to have been resolved [according to GitHub Status](https://www.githubstatus.com/incidents/m70hk23gx3nx). We should expect it to run in at most a few hours.

**Reply by fbricon** [reply]
nope, job still stuck.

**Reply by InSyncWithFoo** [reply]
([Mine too](https://github.com/InSyncWithFoo/pyright-langserver-for-pycharm/actions/runs/10000928969).) I guess the new version will have to be published "manually"?

**Comment by fbricon** [comment]
LSP4IJ 0.3.0 is now available on the JetBrains Marketplace: https://plugins.jetbrains.com/plugin/23257-lsp4ij/edit/versions/stable/574364

Enjoy!
